### DISCUSSION TITLE
Keeping scripts up to date with CompVis main

### DISCUSSION THREAD

**Discussion by JustMaier** [hlky/stable-diffusion#143] [General]
Hey @hlky I noticed that your awesome package here includes a version of `txt2img` from the initial release of stable-diffusion, it seems like it's relatively easy to drop in the replacement scripts, but I was wondering if you had a preferred method or plan for keeping this in sync with the main fork.

**Comment by hlky** [comment]
I suppose I could just sync those versions of the scripts, some of the others include k-diffusion etc
`webui.py` has the `--cli` option which takes input from `txt2img.yaml` and `img2img.yaml`

**Reply by JustMaier** [reply]
Yeah, after looking at things more carefully, It seems that webui isn't even using the `txt2img.py` and instead has it's own implementation... Is that right?
